sympy/solvers/inequalities.py

```diff
@@ -401,7 +401,14 @@ def solve_univariate_inequality(expr, gen, relational=True):
     e = expr.lhs - expr.rhs
     parts = n, d = e.as_numer_denom()
     if all(i.is_polynomial(gen) for i in parts):
-        solns = solve(n, gen, check=False)
+        if S.Infinity in n.args or S.NegativeInfinity in n.args:
+            pop1 = Symbol('pop1')
+            pop2 = Symbol('pop2')
```

Comment:
I think you should use dummy symbols here, otherwise it won't work if the user has a symbol called `pop1` in the expression, will it?